web/modules/custom/contact_information/tests/src/ExistingSite/ContactInformationWorkflowTest.php

```diff
@@ -85,15 +85,12 @@ public function testWorkflow(): void {
         ]);
 
         // Override the user to be checked for the allowed transitions.
-        $actual_transitions = $this->workflowHelper->getAvailableTransitions($content, $this->$user_var);
-        $actual_transitions = array_map(function ($transition) {
-          return $transition->getId();
-        }, $actual_transitions);
+        $actual_states = $this->workflowHelper->getAvailableStates($content, $this->$user_var);
 
-        sort($actual_transitions);
-        sort($transitions);
+        sort($actual_states);
```

Comment:
In general I highly personally dislike the idea to call these `actual/expected_states` and not `actual/expected_target_states`. I know it is only one current state but given that the workflows are working with multiple `from` states to a single `target` state, in the future, for the dev that will have to get down with it, it might be quite confusing.. I would suggest to change the suffix to all these cases to either `_target_states` or `_to_states` (the `to` keyword is the one the module uses) to both the variables and the comments..